interface/grpc/service/service.go

```diff
@@ -30,7 +30,7 @@ func (s *Server) EmitEvent(context context.Context, request *serviceapi.EmitEven
 
 // ListenTask creates a stream that will send data for every task to execute.
 func (s *Server) ListenTask(request *serviceapi.ListenTaskRequest, stream serviceapi.Service_ListenTaskServer) error {
-	ln, err := s.api.ListenTask(request.Token)
+	ln, err := s.api.ListenExecution(request.Token, nil)
```

Comment:
Shouldn't it listen only for "Created/InProgress" execution?

Reply:
I get the following error when running a service with `service dev` and calling multiple time `service execute`:
```
service | (node:1) UnhandledPromiseRejectionWarning: Error: 2 UNKNOWN: Execution status error: "completed" instead of "in progress"
```

to reproduce:

In one terminal:
```
mesg-core service dev https://github.com/mesg-foundation/service-js-function --event-filter=""
```

in an other:
```
mesg-core service execute js-function --task execute --data code="(a) => a + ' michel'" --data inputs='hello'
// no error
mesg-core service execute js-function --task execute --data code="(a) => a + ' michel'" --data inputs='hello'
// error in log of dev command
```